js/data.js

```diff
@@ -436,7 +491,7 @@ var data = {
         kennung: [ITSE, HCT, SAMT],
         lehrform: ["Seminar", "Projekt"],
         modul: ["IT Systems Engineering", "Vertiefungsgebiete"],
-        semester: ss14,
+        semester: [ss14, ss15],
```

Comment:
I think the HCI way of doing things is to create a seminar with a slightly different name each semester, so that one can re-attend them each semester. So we would need to make this an extra course. Probably just name it hciProject1SS15, or sth. like that.